349.IntersectionofTwoArrays/set_intersection.cpp

```diff
@@ -0,0 +1,16 @@
+class Solution {
+public:
+    vector<int> intersection(vector<int>& number1, vector<int>& number2) {
+        set<int> unique_number1;
+        for (auto i : number1) {
+            unique_number1.insert(i);
+        }
```

Comment:
```suggestion
        set<int> unique_number1(number1.begin(), number1.end());
```

こうも書けますね
https://en.cppreference.com/w/cpp/container/set/set

Reply:
@fhiyo 
レビューありがとうございます。
この書き方知りませんでした。こちらの方がスッキリかけますね。step4.cppに追加しました。